packages/tcplay-veracrypt/build.sh

```diff
@@ -0,0 +1,7 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/veracrypt/tc-play
+TERMUX_PKG_DESCRIPTION="Free and simple TrueCrypt implementation based on dm-crypt."
+TERMUX_PKG_LICENSE="BSD 3-Clause"
+TERMUX_PKG_SRCURL=https://github.com/veracrypt/tc-play.git
+TERMUX_PKG_GIT_BRANCH=master
```

Comment:
Please build from the v2.0 tag since that is the specified version (TERMUX_PKG_GIT_BRANCH does not have to be set in that case). 
Alternatively you could set the version to 2.0-git, and specify which commit to build from. We need some way to keep track of which commit/version we are building anyways

Reply:
The latest tag is too old. Using source zip now.